.github/dependabot.yml

```diff
@@ -137,6 +137,24 @@ updates:
     schedule:
       interval: "weekly"
       day: "sunday"
+  - package-ecosystem: "gomod"
+      directory: "/instrumentation/github.com/aws/aws-sdk-go-v2/otelaws"
```

Comment:
It is instrumenting aws services such as 
```
"github.com/aws/aws-sdk-go-v2/service/dynamodb"
"github.com/aws/aws-sdk-go-v2/service/route53"
"github.com/aws/aws-sdk-go-v2/service/s3"
```

Reply:
Based on [the example](https://github.com/open-telemetry/opentelemetry-go-contrib/pull/621/files#diff-3332dda08f692552d38117f849c2144d5312c85724dbac39966b19e4e99a3131R74) this is instrumenting the `github.com/aws/aws-sdk-go-v2/config` package. Is it used to instrument another package? If not, calling this `github.com/aws/aws-sdk-go-v2/config/otelconfig` will conform with our naming policy.

Reply:
It's complicated.  The `github.com/aws/aws-sdk-go-v2/config` package is where the instrumentation is injected into the AWS SDK, but the instrumentation is general and serves to instrument a variety of packages underneath the `github.com/aws/aws-sdk-go-v2/*` namespace.  I think the current naming is appropriate since the convention as applied to that namespace would then require `instrumentation/github.com/aws/aws-sdk-go-v2/otelaws-sdk-go-v2` and that's just overkill! :)

Reply:
Event it fits nameing rule, but `github.com/aws/aws-sdk-go-v2/config/otelconfig` sounds not good since we are instrumenting aws clients :)